<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createPopUp</class>
 <widget class="QDialog" name="createPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:negro_fondo.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>7770</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>400</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SELECCIONAR MAPA PARA COMENZAR EDICIÃ“N</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="labelName">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>344</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nombre del mapa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>INGRESAR NOMBRE MAPA</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>291</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:noche.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="nightButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>NOCHE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="snowButton">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>0</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>NIEVE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>291</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:snow.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>30</y>
          <width>291</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:dia.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="snowButton_2">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>0</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>DIA</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
